src/api/auth-identities.tsx

```diff
@@ -107,14 +107,16 @@ export const deleteIdentities = async (
   });
 };
 
-export const createFineGrainedTlsIdentity = async (
-  clientName: string,
-): Promise<TlsIdentityTokenDetail> => {
+export const createFineGrainedTlsIdentity = async (identity: {
+  name: string;
+  groups: string[];
+}): Promise<TlsIdentityTokenDetail> => {
```

Comment:
```suggestion
export const createFineGrainedTlsIdentity = async (name: string, groups: string[]): Promise<TlsIdentityTokenDetail> => {
```
Maybe a bit simpler to unpack this outside of the api layer?